dask/dataframe/core.py

```diff
@@ -1026,6 +1026,10 @@ def elemwise(op, *args, **kwargs):
 
     if other:
         op2 = partial_by_order(op, other)
+        other_argname = [o[1] for o in other]
+        if len(other_argname) <= 1:
+            other_argname = other_argname[0]
+        op2.__name__ = '{0}({1})'.format(op.__name__, other_argname)
```

Comment:
This change should probably happen within the `partial_by_order function` so that arrays can benefit from it as well.  We should probably also add a `, ...` at the end.